pkg/kube/pod.go

```diff
@@ -349,3 +354,17 @@ func strategicMergeJsonPatch(original, override interface{}) ([]byte, error) {
 	}
 	return mergedPatch, nil
 }
+
+// getPodStartTimeout returns the pod ready wait timeout from ENV if configured
+// returns the default of 15 minutes otherwise
+func getPodStartTimeout() time.Duration {
```

Comment:
rename to getPodReadyWaitTimeout based on the previous comment about the variable names ?